parquet-hadoop/src/main/java/parquet/hadoop/ParquetInputFormat.java

```diff
@@ -232,14 +240,36 @@ public ParquetInputFormat() {
   /**
    * @param configuration to find the configuration for the read support
    * @return the configured read support
+   * @deprecated use getReadSupportInstance static methods instead
```

Comment:
I don't see any other uses of this class when I grep through the code. Did you mean outside this repository? The reference in `mapred.DeprecatedParquetInputFormat` has been updated to use `getReadSupportInstance(Configuration)`.